app/api/dao/user.py

```diff
@@ -156,6 +165,15 @@ def change_password(user_id, data):
 
         return {"message": "Current password is incorrect."}, 400
 
+    @staticmethod
+    def retrieve_profile_picture(user_id, data):
+
+        user = UserModel.find_by_id(user_id)
+        if user.photo_url is not None:
+            return user.photo_url, 200
+
+        return {"message": "User has not updated profile picture"}, 400
```

Comment:
Now that you're other PR was merged can you make this message a constant ?

Reply:
@isabelcosta Sure :)